packages/map-template/src/components/Wayfinding/Wayfinding.scss

```diff
@@ -26,8 +26,20 @@
         position: absolute;
         top: var(--spacing-x-small);
         right: var(--spacing-x-small);
-        background: none;
-        border: none;
+        background-color: var(--tailwind-colors-gray-100);
+        border: 1px solid var(--tailwind-colors-gray-200);
+        margin-left: auto;
+        width: var(--spacing-xx-large);
+        height: var(--spacing-xx-large);
+        border-radius: var(--rounding-large);
+        display: grid;
+        justify-content: center;
+        align-items: center;
+
+        &:hover {
+            background-color: var(--tailwind-colors-gray-200);
+            cursor: pointer;
```

Comment:
I know @markhougaard has opinions on how to style the cursor on buttons. Any input on this being a pointer?

Reply:
So when hovering the buttons, should they be default or pointer?

Reply:
@andreeaceachir142 Ah, `default` _cursor_: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
Alright. I'll change it in all the instances.